현호/11주차_11725.js

```diff
@@ -0,0 +1,43 @@
+const fs = require("fs");
+const input = fs.readFileSync("/dev/stdin").toString().trim().split("\n");
+const size = Number(input.shift());
+
+function solve() {
+  // 트리 생성
+  const tree = Array.from({ length: size + 1 }, () => []);
+  // 방문 배열 생성
+  const visited = Array.from({ length: size+ 1 }, () => 0);
```

Comment:
```suggestion
  const visited = Array.from({ length: size + 1 }, () => 0);
```
7번째 줄과는 다르게 연산자 앞에 띄어쓰기가 안되어 있어서 한 줄 남겨봅니다!!

Reply:
이런세상에 ㅋㅋ 감사합니다